sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -290,11 +306,12 @@ def __init__(self, fully_qualified_namespace, eventhub_name, credential, **kwarg
             self._credential = credential  # type: ignore
         self._keep_alive = kwargs.get("keep_alive", 30)
         self._auto_reconnect = kwargs.get("auto_reconnect", True)
-        self._mgmt_target = "amqps://{}/{}".format(
```

Comment:
was mgmt_target no longer needed? Don't see it re-added after the changes below

Reply:
yeah. This is only being used by uamqp and only in one place (mgmt client creation) so I re-constructed it there.